scripts/test-sim-benchmark.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+set -ueo pipefail
+
+# Purpose: This script executes uplink upload and download benchmark tests against storj-sim.
+# Setup: Setup and run storj-sim. Remove existing uplink configs.
+# Usage: from the root of storj dir, run ./scripts/test-sim-benchmark.sh
+
+# create an api key that the uplink will use
+regEndpoint=http://127.0.0.1:10002/registrationToken/?projectsLimit=1;
+endpoint=http://127.0.0.1:10002/api/graphql/v0;
+satellitePublicGRPC=http://127.0.0.1:10000;
+
+email=`date +%s`@email.com;
+projectName=project`date +%s`;
+
+# In order to create an api key, do the following 5 steps:
+# 1. create reg token
+secret=$(curl -X GET -H "Authorization: secure_token" $regEndpoint 2>/dev/null | awk -F'"' '$2=="secret"{print $4}');
```

Comment:
`$regEndpoint` has a `?` character contained in it, which the shell treats as a special character (the one-character wildcard). Under default conditions, Bash does pathname expansion after parameter expansion, so if it happens that the value in `$regEndpoint` matches a filename in the current directory (maybe unlikely, sure), things are going to get really weird. My point is, I highly recommend double-quoting variables like this—_always_—unless you are quite certain that you actually want it to be expanded as multiple words, or for special shell characters in the variable's value to be treated specially by the shell.

Reply:
thanks for explaining that. I will switch to double quoting